joomla/joomla-cms
# [4.0] Complete list item clickable (Sytem & Help)

Pull Request for Issue # .

### Summary of Changes
In the Help and System pages, this PR makes the entire list item clickable rather than just the text.

### Testing Instructions
Apply PR navigate to Help or System. Full items should be clickable rather than just the text.
